e2etests/testdata/forbidden-annotation-config.yaml

```diff
@@ -1,13 +1,24 @@
 checks:
   addAllBuiltIn: false
 customChecks:
-  name: "forbid-annotation-reloader-stakater-auto"
-  description: ""
-  remediation: "Remove reloader.stakater.com/auto annotation"
-  scope:
-    objectKinds:
-      - DeploymentLike
-  template: "forbidden-annotation"
-  params:
-    key: "reloader.stakater.com/auto"
-    value: 'true'
+  - name: "forbid-annotation-reloader-stakater-auto"
+    description: ""
+    remediation: "Remove reloader.stakater.com/auto annotation"
+    scope:
+      objectKinds:
+        - DeploymentLike
+    template: "forbidden-annotation"
+    params:
+      key: "reloader.stakater.com/auto"
+      value: 'true'
+  - name: invalid-irsa-role
+    description: "IRSA annotations must have a valid IAM Role ARN value"
+    remediation: "Validate the format of the annotation's value to ensure it is a valid IAM Role ARN"
+    scope:
+      objectKinds:
+        - ServiceAccount
+    template: "forbidden-annotation"
+    params:
+      key: "eks.amazonaws.com/role-arn"
+      value: |
+        !arn:aws:iam::\d{12}:role\/[a-zA-Z_+=,.@-]+$
```

Comment:
Where this regexp come from?

Reply:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html#:~:text=%3A%20No%20interruption-,RoleName,-A%20name%20for

Reply:
> [RoleName is] a string of characters consisting of upper and lowercase alphanumeric characters with no spaces. You can also include any of the following characters: _+=,.@-.

Your regexp does not allow numeric values

I think role name part should be `[\w+=,.@-]{1,64}` as defined in [CreateRole](https://docs.aws.amazon.com/IAM/latest/APIReference/API_CreateRole.html)

Reply:
Awesome, thanks for the feedback and reference. I'll make adjustments